content/zh/docs/reference/access-authn-authz/extensible-admission-controllers.md

```diff
@@ -633,7 +633,7 @@ Example of a minimal response from a webhook to allow a request:
 <!--
 Example of a minimal response from a webhook to forbid a request:
```

Comment:
so the above line English text is wrong?

Reply:
yes, the English text is wrong too. I did not check the English text, it has been corrected in the PR.

Reply:
Thanks for the confirmation.